### ISSUE TITLE
Invalid generated TS type on schema with param case elements/attributes

### ISSUE THREAD

**Issue by darcien** [dderevjanik/wsdl-tsclient#18]
TL;DR when the schema have param case (like-this), the generated TS types will be invalid.

Unfortunately I'm not allowed to share the full WSDL schema.
But I can provide an example.

For example, with this complex type:
![image](https://user-images.githubusercontent.com/19742419/123629400-73f5e980-d83e-11eb-8862-f18ea4941b0e.png)

The generated TS will look like this:
![image](https://user-images.githubusercontent.com/19742419/123629449-7e17e800-d83e-11eb-962b-6f2166ba9222.png)

To make the generated TS valid, we just need to wrap the object property with quote mark like:
![image](https://user-images.githubusercontent.com/19742419/123629628-b15a7700-d83e-11eb-9530-cde95b5ac2b9.png)

This issue is happening since v1.2.0.

This doesn't hapeen in v1.1.4.

Might be related to the changes happening in https://github.com/dderevjanik/wsdl-tsclient/pull/17

Thanks for the great package by the way, you guys rock 🤘

**Comment by dderevjanik** [comment]
Hi, thanks !

I checked #17 and it just reveals bug. Until now, every property name was camelcased, which is wrong ! It'll throw an error in runtime. So, instead of camelcasing property name, it should prefix all generated property names with `"` as in your screenshot.

**Comment by dderevjanik** [comment]
Weird thing that ts-morph doesn't automatically generates `"` for property name, which consist from `-`
source: https://github.com/dsherret/ts-morph/issues/1160

**Comment by dderevjanik** [comment]
@darcien @starscoccer please, could you test latest dev version of wsdl-tsclient with your wsdl files ? 

`npm i -g wsdl-tsclient@dev` should install `wsdl-tsclient@1.3.1-alpha.0` - this version should work with `Human_Resource.wsdl` file.

**Comment by darcien** [comment]
Works for me now, thanks for fixing this quickly ♥️
